contents/docs/self-host/_snippets/sunset-disclaimer.mdx

```diff
@@ -0,0 +1,18 @@
+<blockquote>
+<p>🌇 <b>Sunsetting Kubernetes Deployments</b></p>
+
+<p>
+This page covers our PostHog Kubernetes deployment, which we are currently in the <a href="/blog/sunsetting-helm-support-posthog">process of sunsetting</a>.
+Existing customers will receive support until May 31, 2023 and we will continue to provide security updates for the next year.
+</p>
+
+<details>
+<summary class="text-sm font-bold">For existing customers</summary>
+<div>We highly recommend migrating to PostHog Cloud (US or EU). Take a look at <a href="/docs/migrate/migrate-to-cloud">this guide</a> for more information on the migration process.</div>
+</details>
+
+<details>
+<summary class="text-sm font-bold">Looking to continue self-hosting?</summary>
+<div>For low volumes we still provide active support for our Open-source Docker Compose deployment. Instructions for deploying can be found <a href="/docs/self-host/open-source/deployment">here</a>.</div>
```

Comment:
Not sure "active support" is the right phrase here. I guess the point is we support hosting it and will fix issues, but we're not going to help people debug their setup.

Reply:
```suggestion
<div>We still support our Open-source Docker Compose deployment for use at low volumes. Instructions for deploying can be found <a href="/docs/self-host/open-source/deployment">here</a>.</div>
```

Thoughts @pjhul ?

Reply:
Totally agree 👍